proto/regen/ecocredit/v1beta1/marketplace/state.proto

```diff
@@ -0,0 +1,98 @@
+syntax = "proto3";
+
+package regen.ecocredit.v1beta1.marketplace;
+
+import "cosmos/orm/v1alpha1/orm.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+// SellOrder represents the information for a sell order.
+message SellOrder {
+
+    option (cosmos.orm.v1alpha1.table) = {
+        id: 1,
+        primary_key: {fields: "order_id"}
+        index: {id: 1 fields: "batch_denom"}
+        index: {id: 2 fields: "owner"}
+    };
+
+    // order_id is the unique ID of sell order.
+    uint64 order_id = 1;
+
+    // owner is the address of the owner of the credits being sold.
+    string owner = 2;
+
+    // batch_denom is the credit batch being sold.
+    string batch_denom = 3;
+
+    // quantity is the quantity of credits being sold.
+    string quantity = 4;
+
+    // ask_price is the price the seller is asking for each unit of the
+    // batch_denom. Each credit unit of the batch will be sold for at least the
+    // ask_price or more.
+    cosmos.base.v1beta1.Coin ask_price = 5;
+
+    // disable_auto_retire disables auto-retirement of credits which allows a
+    // buyer to disable auto-retirement in their buy order enabling them to
+    // resell the credits to another buyer.
+    bool disable_auto_retire = 6;
+}
+
+// BuyOrder represents the information for a buy order.
+message BuyOrder {
+
+    option (cosmos.orm.v1alpha1.table) = {
+        id: 2,
+        primary_key: {fields: "buy_order_id"}
+        index: {id: 1 fields: "buyer"}
+    };
```

Comment:
Following #671, we need to include the expiration message field and add an expiration index.

Reply:
https://github.com/regen-network/regen-ledger/pull/700/commits/22491e9d5d575fadaa875d201750bf95933ea5c9

Reply:
Same here..